src/idl/src/tree.c

```diff
@@ -1383,6 +1385,22 @@ bool idl_is_base_type(const void *node)
   return true;
 }
 
+bool idl_is_const(const void *node)
+{
+  if (!idl_is_masked(node, IDL_CONST))
+  {
+    return false;
+#ifndef NDEBUG
+  }
+  else
+  {
+    const idl_const_type_t * const_type = ((const idl_const_t *) node)->type_spec;
+    assert(idl_is_base_type(const_type) || idl_is_string(const_type));
+#endif
+  }
+  return true;
+}
```

Comment:
This check is incorrect. It can also be an enumerator.

Reply:
@k0ekk0ek: Thx for catching that one, must have slipped my mind. Added enumerator and updated PR accordingly.